client/js/scoreboard.js

```diff
@@ -32,9 +32,12 @@ export default class Scoreboard {
 		return li;
 	}
 
-	updateScores(players) {
+	updateScores(players, board) {
 		players.forEach((player) => {
 			this.playersToLi.get(player).find('span.score').text(player.score);
+			if(player === board.current){
+				this.playersToLi.get(player).addClass('current');
```

Comment:
👍 
Beware, you're facing a merge problem given recent changes by @matthieu-brillaxis ;-)
sync up with him.